apps/ledger-live-desktop/src/renderer/screens/settings/sections/General/WalletSyncRow.tsx

```diff
@@ -13,7 +88,6 @@ const SideContentWalletSync = () => {
 
 const WalletSyncRow = () => {
   const lldWalletSync = useFeature("lldWalletSync");
```

Comment:
I don't understand why we use this FF here. Because here I need to activate the feature flag to see the entrypoint in the settings, so it's the `SideContentWalletSync` that it's displayed. Could you update allowing us to test what you did, please?

Reply:
const lldWalletSync = useFeature("lldWalletSync") should be used to display the row in the settings only yes :)